ui/tests/integration/util/exec-socket-xterm-adapter-test.js

```diff
@@ -30,7 +90,7 @@ module('Integration | Utility | exec-socket-xterm-adapter', function(hooks) {
       },
     });
 
-    new ExecSocketXtermAdapter(terminal, mockSocket);
+    new ExecSocketXtermAdapter(terminal, mockSocket, '');
```

Comment:
Point of interest: the xtreem dynamism of Javascript means that this isn’t needed, it could just be left off and amount to `undefined`. However, I think it’s fine to keep just to be explicit that three arguments are expected.